plugin.video.tv3.cat/resources/language/resource.language.ca_es/strings.po

```diff
@@ -0,0 +1,125 @@
+# Kodi Media Center language file
```

Comment:
Language folder layout is not compatible with Gotham but only Isengard and onwards. Please keep the old folder structure but use the new .po files

Reply:
Changed to old folder structure with new .po files. Thanks